cfme/infrastructure/virtual_machines.py

```diff
@@ -151,6 +215,47 @@ class InfraVmReconfigureView(BaseLoggedInPage):
     is_displayed = False
 
 
+class MigrateView(BaseLoggedInPage):
+    title = Text('#explorer_title_text')
+
+    @View.nested
+    class form(View):  # noqa
+        submit_button = Button('Submit')
+        cancel_button = Button('Cancel')
+
+        @View.nested
+        class request(Tab):  # noqa
+            TAB_NAME = 'Request'
+            email = WInput(name='requester__owner_email')
+            first_name = WInput(name='requester__owner_first_name')
+            last_name = WInput(name='requester__owner_last_name')
+            notes = WInput(name='requester__request_notes')
+            manager_name = WInput(name='requester__owner_manager')
+
+        @View.nested
+        class environment(Tab):  # noqa
+            TAB_NAME = 'Environment'
+            # Infra
+            datacenter = BootstrapSelect('environment__placement_dc_name')
+            cluster = BootstrapSelect('environment__placement_cluster_name')
+            resource_pool = BootstrapSelect('environment__placement_rp_name')
+            folder = BootstrapSelect('environment__placement_folder_name')
+            host_filter = BootstrapSelect('environment__host_filter')
+            host_name = Table('//div[@id="prov_host_div"]/table')
+            datastore_filter = BootstrapSelect('environment__ds_filter')
+            datastore_name = Table('//div[@id="prov_ds_div"]/table')
+
+        @View.nested
+        class schedule(Tab):  # noqa
+            TAB_NAME = 'Schedule'
+            # TODO radio widget # schedule_type = Radio('schedule__schedule_type')
```

Comment:
https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L1416

Aware of widgetastic_manageiq.RadioGroup and just postponing implementation?